en/includes/deploy/security/keystores/configure-custom-keystores.md

```diff
@@ -0,0 +1,125 @@
+# Configure Custom KeyStores for Authentication Protocols
+
+By default, WSO2 Identity Server uses the private key and certificate from the Primary KeyStore in all authentication protocols such as,
```

Comment:
```suggestion
By default, WSO2 Identity Server uses the private key and certificate from the primary keystore in all authentication protocols such as,
```

Reply:
addressed by https://github.com/wso2/docs-is/pull/5297/commits/150dc2057392db4922a85d4973fcba77dab36f59